exhibitor.conf.tmpl

```diff
@@ -11,3 +11,4 @@ election-port=3888
 zoo-cfg-extra=tickTime\=2000&initLimit\=10&syncLimit\=5&quorumListenOnAllIPs\=true
 auto-manage-instances-settling-period-ms=0
 auto-manage-instances=1
+java-environment=#!/bin/bash \nexport JAVA_TOOL_OPTIONS="-javaagent:/opt/jolokia-jvm-${JOLOKIA_VERSION}-agent.jar=port=8779,host=0.0.0.0"
```

Comment:
Can you please explain what is java-environment and how this thing is used? I failed to find documentation for it

Reply:
The value of java-environment is placed in Config -> Ensemble -> java.env script. It is "Script to write as the “java.env” file which gets executed as part of the ZooKeeper start script. Optional – leave blank for no file." [docs](https://github.com/soabase/exhibitor/wiki/Configuration-UI)